docs/api/instruments/heidenhain/index.rst

```diff
@@ -0,0 +1,14 @@
+.. module:: pymeasure.instruments.heidenhain
+
```

Comment:
this warning above means that you need to add a reference in  docs/api/instruments/index.rst

Reply:
Oops. Fixed that.